iothub/service/src/AmqpServiceClient.cs

```diff
@@ -123,10 +128,7 @@ public async override Task SendAsync(string deviceId, Message message, TimeSpan?
                     .SendMessageAsync(amqpMessage, IotHubConnection.GetNextDeliveryTag(ref _sendingDeliveryTag), AmqpConstants.NullBinary, timeout.Value)
                     .ConfigureAwait(false);
 
-                if (Logging.IsEnabled)
-                {
-                    Logging.Info(this, $"Outcome was: {outcome?.DescriptorName}", nameof(SendAsync));
-                }
+                Logging.Info(this, $"Outcome was: {outcome?.DescriptorName}", nameof(SendAsync));
```

Comment:
surround with square brackets in case it is null

Reply:
Actually null is already logged with braces, so I left it as is: https://github.com/Azure/azure-iot-sdk-csharp/blob/azabbasi/AmqpServiceClientLogs/common/src/Logging.Common.cs#L44

Reply:
interesting